### ISSUE TITLE
Quick Start guide at the top of readme.md

### ISSUE THREAD

**Issue by KirillOsenkov** [SergeyTeplyakov/ObjectLayoutInspector#41]
Imagine I need to quickly print the layout of a type - I need to add the PackageReference to the NuGet and add some code to a test.

Would be great to have a super brief getting started guide at the very top that shows which NuGet package to add and what sample code to write.

@SergeyTeplyakov

**Comment by SergeyTeplyakov** [comment]
Agree. I need it even for myself. I keep forgetting what should I use.
